hangman.py

```diff
@@ -83,63 +77,170 @@ def get_available_letters(letters_guessed):
     returns: string (of letters), comprised of letters that represents which letters have not
       yet been guessed.
     '''
-    # FILL IN YOUR CODE HERE AND DELETE "pass"
-    pass
-    
-    
+    letters = list(string.ascii_lowercase)
+    result = ""
+    for i in letters_guessed:
+        letters.remove(i)
+    for i in letters:
+        result += i
+    return result
+
+
+def check_warnings(warnings, user_guess, repeated_guesses, show_word):
+    '''
+
+    :param warnings: warnings that are given to user
+    :param user_guess: user's input
+    :param repeated_guesses: letters that have been repeated
+    :param show_word: output that is visible to user
+    :return: warnings that are left
+    '''
+    warnings -= 1
+    if not user_guess.isalpha():
+        print('Oops! That is not a valid letter. You have ' + str(warnings) + ' warnings left: ' + show_word)
+    elif user_guess in repeated_guesses:
+        print('Oops! You\'ve already guessed that letter. You have ' + str(warnings) + ' warnings left. ' + show_word)
+    print('-----------------')
+    return warnings
+
+
+def check_guesses(guesses, user_guess, repeated_guesses, show_word):
+    '''
+
+    :param guesses: guesses that are given to user
+    :param user_guess: user's input
+    :param repeated_guesses: letters that have been repeated
+    :param show_word: output that is visible to user
+    :return: warnings that are left
+    '''
+    guesses -= 1
+    if not user_guess.isalpha():
+        print("Oops! That is not a valid letter. You have no warnings left so you lose one guess. " + show_word)
+    elif user_guess in repeated_guesses:
+        print(
+            'Oops! You\'ve already guessed that letter. You have no warnings left so you lose one guess: ' + show_word)
```

Comment:
**Dont repeat yourself.** The same block of code in function `check_warnings`. Fix it.